### PR TITLE
Update Bullguard.com.xml

### PR DESCRIPTION
**Pull request by I-have-50-open-PRs-to-HTTPS-EVERYWHERE**

### CODE REVIEW

File: src/chrome/content/rules/Bullguard.com.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
 	<target host="bullguard.com" />
 	<target host="www.bullguard.com" />
-	<rule from="^http://www\.bullguard\.com/" to="https://www.bullguard.com/"/>
-	<rule from="^http://bullguard\.com/" to="https://www.bullguard.com/"/>
-</ruleset>
 
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http:" to="https:" />
+</ruleset>
```
Comment (J0WI) [inline src/chrome/content/rules/Bullguard.com.xml]: See also:
```
backup.bullguard.com
backup1.bullguard.com
dojo.bullguard.com
download.bullguard.com
htmltest.bullguard.com
iotscanner.bullguard.com
media.bullguard.com
msm.bullguard.com
myaccount.bullguard.com
safebrowsing.bullguard.com
shop.bullguard.com
support.bullguard.com
wwworigin.bullguard.com

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@J0WI Added.

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@J0WI Added.

File: src/chrome/content/rules/Bullguard.com.xml [hunk 2]
```diff
@@ -1,6 +1,24 @@
+<!--
+	Unable to connect:
+		media.bullguard.com
+	Invalid Certificate:
+		safebrowsing.bullguard.com
+		wwworigin.bullguard.com
+	Unable to get Local Issuer certificate:
```
Comment (J0WI) [inline src/chrome/content/rules/Bullguard.com.xml]: Please use something like `incomplete certificate chain` for this kind of issues.

**Comment by I-have-50-open-PRs-to-HTTPS-EVERYWHERE** [comment]
@J0WI Added.

**Pull request merged**